src/index.ts

```diff
@@ -75,13 +76,13 @@ export default function mitt<Events extends Record<EventType, unknown>>(
 		 * @memberOf mitt
 		 */
 		off<Key extends keyof Events>(type: Key, handler?: GenericEventHandler) {
-			const handlers: Array<GenericEventHandler> | undefined = all!.get(type);
+			let handlers: Array<GenericEventHandler> | undefined = all!.get(type);
```

Comment:
This is a non-functional change - it just slightly improves compression gains by making the structure of `off()` match that of `on()`.